src/test/java/com/graphhopper/converter/resource/ConverterResourcePeliasTest.java

```diff
@@ -0,0 +1,71 @@
+package com.graphhopper.converter.resource;
+
+import com.graphhopper.converter.ConverterApplication;
+import com.graphhopper.converter.ConverterConfiguration;
+import com.graphhopper.converter.api.GHResponse;
+import io.dropwizard.client.JerseyClientBuilder;
+import io.dropwizard.testing.ResourceHelpers;
+import io.dropwizard.testing.junit.DropwizardAppRule;
+import org.glassfish.jersey.client.ClientProperties;
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Robin Boldt
+ */
+public class ConverterResourcePeliasTest {
+    @ClassRule
+    public static final DropwizardAppRule<ConverterConfiguration> RULE =
+            new DropwizardAppRule<>(ConverterApplication.class, ResourceHelpers.resourceFilePath("converter.yml"));
+
+    @Test
+    public void testHandleForward() {
+        Client client = new JerseyClientBuilder(RULE.getEnvironment()).build("test forward client");
+
+        client.property(ClientProperties.CONNECT_TIMEOUT, 100000);
+        client.property(ClientProperties.READ_TIMEOUT, 100000);
```

Comment:
Why do we need so high timeout settings?

Reply:
This is only used for tests. I think the reason was that if you are on a slow internet connection and run the tests, they can timeout. IIRC this has happened on Travis at some point as well.

Reply:
BTW: they should be similar for all tests.

Reply:
Makes sense. Thx!